### DISCUSSION TITLE
Arc Browser

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#34] [General]
# Arc Browser

Check if Arc Browser is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/29b1fa02-faec-459d-bc3b-dd8c26fc11b0

**Comment by talynone** [comment]
ARM64 version can be downloaded here:

https://releases.arc.net/windows/prod/1.18.0.48077/Arc.arm64.msix

**Reply by talynone** [reply]
Latest version:

https://releases.arc.net/windows/prod/1.19.2.49737/Arc.arm64.msix

**Reply by OpenSource03** [reply]
Thanks. I’ve updated the app status to beta support for now.
